optimum/exporters/openvino/convert.py

```diff
@@ -389,7 +389,10 @@ def ts_patched_forward(*args, **kwargs):
                 if patch_16bit_model:
                     from openvino.frontend.pytorch.patch_model import __make_16bit_traceable
 
+                    # frontend may riases confusing warnings about modules already patched if model splitted on several parts
+                    logging.disable(logging.WARNING)
                     __make_16bit_traceable(model)
+                    logging.disable(logging.NOTSET)
```

Comment:
Is it possible that with this not only irrelevant warnings will be hidden, but the useful ones too? Is there a way to filter specifically those somehow?

Reply:
this function does not produce any additional warnings